docs/how-to/configure/region-merge.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Region Merge
+summary: Learn how to configure Region Merge in TiKV.
+category: how-to
+---
+
+# Region Merge
+
+TiKV replicates a segment of data in Regions via the Raft status machine. As data writes increase, a Region Split happens when the size of the region or the number of serial keys has reached a threshold. Conversely, if the size of the Region or the amount of keys shrinks because of data deletion, we can use Region Merge to merge adjacent regions that are smaller. This relieves some stress on Raft.
+
+
+## Merge process
+
+Region Merge is initiated by the Placement Driver (PD). The steps are:
+
+1. PD polls the sizes and number of keys of all regions on the TiKV node.
+
+2. When the region size is less than `max-merge-region-size` or the number of keys the region includes is less than `max-merge-region-keys`, PD performs Region Merge on the smaller of the two adjacent Regions.
+
+> **Note:**
+>
+> - All replicas of the merged Regions must be on the same TiKV.
```

Comment:
```suggestion
> - All replicas of the merged Regions must be on the same group of TiKVs.
```
I don't know whether it is a clear description?

Reply:
I think it's better. :) Or maybe "TiKV cluster"?

Reply:
Now using TiKV cluster as Ana suggested. @Connor1996

Reply:
Oh, it's not what I mean. I mean if there are five TiKV instances A, B, C, D, E in one cluster, the replicas of the two Regions involving in Merge should be on the same set of TiKVs. For example,  Region 1 with three peers, p11 on storeA, p12 on storeB, p13 on storeC, can be merged with Region 2 with three peers, p21 on storeC, p22 on storeB, p23 on storeC. But it can't be merged with Region 3 with three peers, p31 on storeA, p32 on storeB, p33 on storeD due to unmatched location.

Reply:
Updated as "same set of TiKVs". Thanks @Connor1996  for your detailed explanation. PTAL again. If it's OK, please approve asap.